<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="0.0000" tests="1" failures="1">
  <testsuite name="Root Suite" timestamp="2021-04-14T07:53:24" tests="0" file="cypress/integration/examples/local_storage.spec.js" time="0.0000" failures="0">
  </testsuite>
  <testsuite name="Local Storage" timestamp="2021-04-14T07:53:24" tests="1" time="0.0000" failures="1">
    <testcase name="Local Storage cy.clearLocalStorage() - clear all data in local storage" time="0.0000" classname="cy.clearLocalStorage() - clear all data in local storage">
      <failure message="Timed out retrying after 4050ms: expected null to equal &apos;red&apos;" type="AssertionError"><![CDATA[AssertionError: Timed out retrying after 4050ms: expected null to equal 'red'
    at Context.eval (https://example.cypress.io/__cypress/tests?p=cypress/integration/examples/local_storage.spec.js:110:48)]]></failure>
    </testcase>
  </testsuite>
</testsuites>